splicemachine/ml/utilities.py

```diff
@@ -262,52 +262,51 @@ def visualize(
             model,
             feature_column_names,
             label_names,
-            tree_name,
+            size=None,
+            horizontal=False,
+            tree_name='tree',
             visual=False,
     ):
         """
         Visualize a decision tree, either in a code like format, or graphviz
         :param model: the fitted decision tree classifier
-        :param feature_column_names: column names for features
-        :param label_names: labels vector (below avg, above avg)
+        :param feature_column_names: (List[str]) column names for features
```

Comment:
Just a minor unimportant comment - the comments / documentation does not have the size parameter.

Reply:
added